.github/workflows/compile.yml

```diff
@@ -7,6 +7,16 @@ on:
       - "*"
   pull_request: {}
 jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
```

Comment:
This needs to run on macos, I think golangci-lint attempts to build some code, which can only be done on a macos runner for vfkit (see the CI failures)

Reply:
I switched this to `macos-latest`